### PR TITLE
HTML doc updates

### PR DESCRIPTION
**Pull request by hssyoo**
Catchall PR for some minor HTML doc updates.

### CODE REVIEW

File: doc/source/guzzle_sphinx_theme/guzzle_sphinx_theme/static/guzzle.css_t [hunk 1]
```diff
@@ -102,6 +111,10 @@ div.highlight {
   font-weight: bold;
 }
 
+#nav-link {
```
Comment (stealthycoin) [inline doc/source/guzzle_sphinx_theme/guzzle_sphinx_theme/static/guzzle.css_t:114]: Lets change this to a class `.nav-link`. id is a unique identifier for a particular html element.

File: requirements-docs.txt [hunk 2]
```diff
@@ -4,7 +4,7 @@ jinja2<3.1
 
 
 docutils>=0.10,<0.17
-Sphinx==1.3.2
+Sphinx==1.4.9
```
Comment (stealthycoin) [inline requirements-docs.txt:7]: What is this bump for?

Comment (hssyoo) [reply requirements-docs.txt:7]: Setting the document language was introduced in 1.4.0
https://www.sphinx-doc.org/en/master/changes.html#:~:text=Add%20%E2%80%9Clanguage%E2%80%9D%20attribute%20to%20html%20tag%20in%20the%20%E2%80%9Cbasic%E2%80%9D%20theme

Comment (stealthycoin) [reply requirements-docs.txt:7]: Got it. That's what I assumed it as based on the commit, but I didn't see it in the changelog.

**Review by stealthycoin — APPROVED** [review]

**Pull request merged**